cedar-policy-validator/src/schema/entity_type.rs

```diff
@@ -193,28 +187,41 @@ impl From<&proto::ValidatorEntityType> for ValidatorEntityType {
     // PANIC SAFETY: experimental feature
     #[allow(clippy::expect_used)]
     fn from(v: &proto::ValidatorEntityType) -> Self {
-        let tags = match &v.tags {
-            Some(tags) => tags.optional_type.as_ref().map(Type::from),
-            None => None,
-        };
+        let name = ast::EntityType::from(
+            v.name
+                .as_ref()
+                .expect("`as_ref()` for field that should exist"),
+        );
+        let descendants = v.descendants.iter().map(ast::EntityType::from).collect();
         Self {
-            name: ast::EntityType::from(
-                v.name
-                    .as_ref()
-                    .expect("`as_ref()` for field that should exist"),
-            ),
-            descendants: v.descendants.iter().map(ast::EntityType::from).collect(),
-            kind: ValidatorEntityTypeKind::Standard(StandardValidatorEntityType {
-                attributes: Attributes::from(
-                    v.attributes
-                        .as_ref()
-                        .expect("`as_ref()` for field that should exist"),
+            name,
+            descendants,
+            // We use emptiness of the `enums` field as an indictor to tell if `v` represents an enumerated entity type or not
+            // In other words, we essentially ignore other fields like `attributes` when `enums` is empty
```

Comment:
I think you dropped a "not". At least this makes more sense to me
```suggestion
            // In other words, we essentially ignore other fields like `attributes` when `enums` is not empty
```